test/models/employment/AccommodationRelocationModelSpec.scala

```diff
@@ -61,7 +61,7 @@ class AccommodationRelocationModelSpec extends UnitTest {
     "return call to AccommodationRelocationBenefitsController when accommodationQuestion and accommodation not defined" in {
```

Comment:
test name can be changed from AccommodationRelocationBenefitsController to LivingAccommodationBenefitAmountController